elementary/monitor/dbt_project/macros/get_resources_latest_invocation.sql

```diff
@@ -0,0 +1,22 @@
+{% macro get_resources_latest_invocation() %}
+  {% set dbt_run_results = ref('dbt_run_results') %}
+  {%- if elementary.relation_exists(dbt_run_results) -%}
```

Comment:
I think that there is no need to check if this table exists.
If a user ran elementary dbt package he should have this table.
Plus I am not sure we want to do nothing if this table does not exist.